Classes/Utility/FocusAreaUtility.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace SomehowDigital\Typo3\MediaProcessing\Utility;
+
+use TYPO3\CMS\Core\Imaging\ImageManipulation\Area;
+
+class FocusAreaUtility
+{
+
+	/**
+	 * @param \TYPO3\CMS\Core\Imaging\ImageManipulation\Area $area
+	 *
+	 * @return float|int
+	 */
+	public static function calculateCenter(float $position, float $size)
+	{
+		// The max method ensures that only positive floats are returned
+		return max(($position + ($size / 2)), 0);
+	}
+}
```

Comment:
Just to be consistent with the codebase, I'd prefer to not have doc blocks. Also, the comment is mostly superfluous due to the line being easy to read, I think.

```suggestion
class FocusAreaUtility
{
	public static function calculateCenter(float $offset, float $size)
	{
		return max(($offset + ($size / 2)), 0);
	}
}
```